@navikt/core/react/src/chat/chat.stories.tsx

```diff
@@ -88,8 +88,8 @@ export const Variants: Story = {
         <React.Fragment key={variant}>
           <h3>{variant}</h3>
           <Chat
-            avatar="NAV"
-            name="NAV"
+            avatar="Ls"
+            name="Ls"
```

Comment:
Any special reason for changing this? Just curious

Reply:
Just since it was used in Figma